conbench/templates/compare-list.html

```diff
@@ -162,15 +147,15 @@ <h6>Top Outliers</h6>
                   {{ c.analysis.pairwise.percent_change }} %
                 </td>
               {% else %}
-                <td></td>
+                <td data-order="99999999">n/a</td>
               {% endif %}
-              {% if c.analysis.lookback_z_score.z_score is not none %}
+              {% if c.analysis.lookback_z_score is not none %}
                 <td class="font-monospace"
                     data-order="{{ c.analysis.lookback_z_score.z_score }}">
                   {{ c.analysis.lookback_z_score.z_score }}
                 </td>
               {% else %}
-                <td></td>
+                <td data-order="99999999">n/a</td>
```

Comment:
This ugly trick sorts non-existant z-score value "highest" in the direction of "improvement", i.e. these rows are last in the table when showing 'regression candidates first'.